pandamium_datapack/data/pandamium/functions/tick_loop.mcfunction

```diff
@@ -6,4 +6,4 @@ execute if predicate pandamium:is_storming run weather rain
 
 function pandamium:misc/toggle_mob_spawning
 
-execute as @a[gamemode=!spectator] if score @s active_particles matches 1.. run function pandamium:misc/particles/tick
+execute as @a[gamemode=!spectator,scores={active_particles=1..}] at @s anchored eyes run function pandamium:misc/particles/tick
```

Comment:
Not all particles are anchored at the player's eyes

Reply:
`anchored` doesn't affect ~ ~ ~
https://minecraft.gamepedia.com/Commands/execute#anchored

Reply:
Ah, thanks 👍